App.js

```diff
@@ -1,21 +1,35 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
-import { NavigationContainer } from '@react-navigation/native'
+import React from "react";
+import { NavigationContainer } from "@react-navigation/native";
+import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
+import { SafeAreaProvider } from "react-native-safe-area-context";
+
+import { Diary, Home } from "./components";
+
+const Tab = createBottomTabNavigator();
 
 export default function App() {
   return (
-    <View style={styles.container}>
-      <Text>my dairy</Text>
-      <StatusBar style="auto" />
-    </View>
+    <SafeAreaProvider>
+      <NavigationContainer>
+        <Tab.Navigator tabBarOptions={{ labelStyle: { fontSize: 20 } }}>
+          <Tab.Screen
+            name="Home"
+            component={Home}
+            options={{
+              headerShown: false,
+              tabBarIcon: () => null,
+            }}
+          />
+          <Tab.Screen
+            name="Diary"
+            component={Diary}
+            options={{
+              headerShown: false,
+              tabBarIcon: () => null,
+            }}
+          />
+        </Tab.Navigator>
```

Comment:
동일 option이 각각의 Screen에 들어간다면 `screenOptions` 를 사용해도 좋을 것 같아요
[공식문서](https://reactnavigation.org/docs/screen-options/#screenoptions-prop-on-group)
```suggestion
        <Tab.Navigator 
          tabBarOptions={{ labelStyle: { fontSize: 20 } }}
          screenOptions={{
              headerShown: false,
              tabBarIcon: () => null,
          }}
        >
          <Tab.Screen
            name="Home"
            component={Home}
          />
          <Tab.Screen
            name="Diary"
            component={Diary}
          />
        </Tab.Navigator>
```

Reply:
Warning이 발생해서 확인해 봤는데 Ian이 말씀해주신 것 처럼 screenOptions이 더 적절해 보여 수정했습니다.🙏